README.md

```diff
@@ -1,8 +1,6 @@
 # linfa
 
-[![Build status](https://travis-ci.org/LukeMathWalker/linfa.svg?branch=master)](https://travis-ci.org/LukeMathWalker/linfa)
-[![Coverage](https://codecov.io/gh/LukeMathWalker/linfa/branch/master/graph/badge.svg)](https://codecov.io/gh/LukeMathWalker/linfa)
-[![Dependencies status](https://deps.rs/repo/github/LukeMathWalker/linfa/status.svg)](https://deps.rs/repo/github/LukeMathWalker/linfa)
+[![Build Status](https://travis-ci.org/rust-ml/linfa.svg?branch=master)](https://travis-ci.org/rust-ml/linfa)
```

Comment:
have you any clue why the dependency status just shows `unknown`? But we can add it later on again

Reply:
No, not really. Just updating the link (https://deps.rs/repo/github/rust-ml/linfa) it says "Failed to analyze repository.
The repository you requested might be structured in an uncommon way that is not yet supported." The maintainer of deps.rs is apparently looking for somebody to take over maintainership.. 

It seems to be a somewhat common issue for popular repositories:

![image](https://user-images.githubusercontent.com/154420/89265307-908ca000-d634-11ea-9797-3aa342b5c9dc.png)

I didn't look beyond this for now.